src/Sarif.Converters/HdfConverter.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using Microsoft.CodeAnalysis.Sarif.Converters.HdfModel;
+
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    public class HdfConverter : ToolFileConverterBase
```

Comment:
>HdfConverter [](http://example.com/codeflow?start=17&length=12)

can you add Hdf here: [https://github.com/microsoft/sarif-sdk/blob/main/src/Sarif.Multitool.Library/ConvertOptions.cs](https://github.com/microsoft/sarif-sdk/blob/main/src/Sarif.Multitool.Library/ConvertOptions.cs)

and here: [https://github.com/microsoft/sarif-sdk/blob/main/src/ReleaseHistory.md](https://github.com/microsoft/sarif-sdk/blob/main/src/ReleaseHistory.md)

add one new line before 2.4.6 (un-released).

Reply:
done both, thanks